### PR TITLE
[WEB-2177] Glossary

### PR DESCRIPTION
**Pull request by bgdeutsch**
### What does this PR do?
adds new glossary

### Motivation
https://datadoghq.atlassian.net/browse/WEB-2177

### Preview 
https://docs-staging.datadoghq.com/brian.deutsch/glossary/glossary/

### Additional Notes
Using archetype template should make it easier to create markdown pages.  Run `hugo new glossary/{term}.md` to produce the markdown file with prefilled frontmatter from the template.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/glossary/terms/tracer.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Tracer
```
Comment (kayayarai) [inline content/en/glossary/terms/tracer.md:2]:
```suggestion
title: tracer
```

File: assets/scripts/components/async-loading.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ import { updateTOC, buildTOCMap } from './table-of-contents';
 import initCodeTabs from './codetabs';
 import { redirectToRegion } from '../region-redirects';
 import { initializeIntegrations } from './integrations';
+import { initializeSecurityRules } from './security-rules';
```
Comment (kayayarai) [inline assets/scripts/components/async-loading.js:5]:
```suggestion
```

File: content/en/glossary/_index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Glossary
```
Comment (kayayarai) [inline content/en/glossary/_index.md:2]:
```suggestion
title: Glossary
is_beta: true
```

**Review by kayayarai — APPROVED** [review]

**Comment by nsollecito** [comment]
Screen Shot 2022-09-15 at 2 57 13 PM

@bgdeutsch the Contact Us button in the footer appears shrunk in this preview

File: content/en/glossary/terms/apm.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: APM
+related_terms:
+    - Tracer
```
Comment (bgdeutsch) [inline content/en/glossary/terms/apm.md:4]: hey @kayayarai - Tracer is a related term here, so there is a link to `#tracer`, but it doesn't seem like there is a definition for tracer yet.

Comment (kayayarai) [reply content/en/glossary/terms/apm.md:4]: oh, let's just take this one out for now then
```suggestion
```

File: content/en/glossary/terms/check.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+---
+title: Check
+synonyms:
+    - APM
+    - Tracer
```
Comment (kayayarai) [inline content/en/glossary/terms/check.md:5]:
```suggestion
```

**Review by nsollecito — APPROVED** [review]

File: content/en/glossary/terms/dogstatsd.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title: DogStatsD
+---
+DogStatsD refers to two related things: a protocol based on StatsD, and an application for reporting metrics which implements that protocol. The DogStatsD protocol is an extension of the StatsD protocol, with some modifications that are specific to the Datadog platform. The DogStatsD application is a service that is bundled with the Agent, and is used as a lightweight mechanism for reporting metrics.
+
+See the <a href='developers/dogstatsd'>DogStatsD documentation</a> for more information.
```
Comment (kayayarai) [inline content/en/glossary/terms/dogstatsd.md:6]:
```suggestion
See the <a href='/developers/dogstatsd'>DogStatsD documentation</a> for more information.
```

**Pull request merged**